addons/music/fnc_getMusicData.sqf

```diff
@@ -0,0 +1,38 @@
+/* ----------------------------------------------------------------------------
+Function: CBA_fnc_getMusicData
+
+Description:
+    A function used to return data from the given music class
+    
+Parameters:
+    one string OR one config path, and a data type (eg 'name')
+    
+Returns:
+    data entry for requested music class
+    
+Example:
+    (begin example)
+        _musicName = ["LeadTrack01_F_Bootcamp","Duration"] call CBA_fnc_getMusicData;
+    (end example)
+
+Author:
+    Fishy
+---------------------------------------------------------------------------- */
+#include "script_component.hpp"
+
+params [["_className",""],["_dataType","name"]];
+private "_config";
```

Comment:
You can change this to `private _config = _classname` and then remove the else from the if statement. If _className is not a config the IS_CONFIG check from line 35 will fail.
That way you can theoretically also omit the isNil check in Line 35. But I'm not sure if one could just pass Nil as _className.